views/views.go

```diff
@@ -89,12 +139,18 @@ type (
 var _ Repo = &Views{}
 
 // New initialises connections, templates, and cookies
-func New(conf Config) *Views {
-	v := Views{}
+func New(conf *Config, host string) *Views {
+	v := &Views{}
 	// Connecting to stores
 	dbStore, err := db.NewStore(conf.DatabaseURL)
 	if err != nil {
-		log.Fatalf("NewStore failed: %+v", err)
+		if conf.Debug {
+			log.Printf("db failed: %+v", err)
+		} else {
+			log.Fatalf("db failed: %+v", err)
+		}
+	} else {
+		log.Printf("connected to db: %s", host)
 	}
```

Comment:
Seems a bit strange for something in the `views` package to be responsible for connecting to the DB - usually you'd pass an existing `Store` instance into `views.New()`. (Again I appreciate that this is pre-existing code, but since you're already changing it around, may as well go the extra mile.)

Reply:
Fair point, will change that now

Reply:
While you're at it, is it worth doing the same for `mailer` - see below